Formula/bazel.rb

```diff
@@ -13,6 +13,7 @@ class Bazel < Formula
 
   depends_on :java => "1.8"
   depends_on :macos => :yosemite
+  depends_on "libomp"
```

Comment:
sure this shouldn't be `libgomp`?

Reply:
When I run the test, it is complaining about not finding 'Library not loaded: /usr/local/opt/libomp/lib/libomp.dylib'. After manually installing it using brew and followed step here https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html#macos, it worked. Not sure if we just brew install, it works or not. Maybe libgomp installs libomp? Or is there actually other part of lightgbm that depends on libgomp which the test cases do not trigger?

Reply:
Might be libgomp on ubuntu but libomp on osx?